plugin-redfish/rfpmodel/Device.go

```diff
@@ -40,19 +40,6 @@ type HTTPHeaders struct {
 	ContentType string `json:"Content-Type"`
 }
 
-//EvtOem ...
-type EvtOem struct {
-	Hpe Hpe `json:"Hpe"`
-}
-
-//Hpe model
```

Comment:
What does this do ? And where is it now?

Reply:
These functions are not being used in plugin-redfish. That's why I removed them.